### ISSUE TITLE
Illegal base64 character 3f

### ISSUE THREAD

**Issue by kenifanying** [nokia/RED#281]
I have some library written in python not automatically discovered by RED. I tried to reload the library but I get the following error:

![default](https://user-images.githubusercontent.com/4479181/53865301-c2f23200-4029-11e9-9718-caa1c3277785.png)

All my python source file has setting the coding to the utf-8 and has the following  line at the top of the file.

` # -*- coding: utf-8 -*-`

I didn't find any strange character at my source file. It's ok to run test case, just can't discovered by RED.

Any assistance would be appreciated.

Python 2.7.15
Robot Framework 3.1.1 (Python 2.7.15 on win32)
RED - Robot Editor version 0.8.11
System: Windows 7 X86_64

**Comment by michalanglart** [comment]
Hi,

please check if you observe the same behavior when preference: 'Robot Framework -> Libraries -> Generate Python libraries libdocs in separate process' is disabled

**Comment by michalanglart** [comment]
Please also tell us:

1) What is the output printed to your console when you're running `libdoc` tool on problematic library? The command that should do it should look somehow similar to this (although the path/name can be different depending on how this library is structured):

`python -m robot.libdoc path\to\your\lib\WebLibrary\PageObject\OnlineUserMngSMP documentation.xml`

2) Does your library print anything to `stdout` (directly or for example via `robot.api.logger`) during module initalization or inside the library constructor (if the library is written as a class)?

**Comment by kenifanying** [comment]
It's ok  when preference: 'Robot Framework -> Libraries -> Generate Python libraries libdocs in separate process' is disabled

**Comment by kenifanying** [comment]
> 
> 
> Please also tell us:
> 
>     1. What is the output printed to your console when you're running `libdoc` tool on problematic library? The command that should do it should look somehow similar to this (although the path/name can be different depending on how this library is structured):
> 
> 
> `python -m robot.libdoc path\to\your\lib\WebLibrary\PageObject\OnlineUserMngSMP documentation.xml`
> 
No strange output, the problematic library is FwLibrary.FwVariables, see below:
![snipaste_20190306_165127](https://user-images.githubusercontent.com/4479181/53868666-7dd1fe00-4031-11e9-8d68-11d0606c4d6e.png)


>     1. Does your library print anything to `stdout` (directly or for example via `robot.api.logger`) during module initalization or inside the library constructor (if the library is written as a class)?

Yes, I use `logger.info` inside the library constructor
```
def fw_log(self,log_info):
        try:
            logger.info(log_info)
        except:
            logger.info(u"fw_log出现打印信息错误!")
```

**Comment by michalanglart** [comment]
Ok, this looks like a problem we've already found. Keep the preference disabled if every other library is recognized and we'll fix the problem in next version

**Comment by michalanglart** [comment]
Fixed in 0.8.12
